checkov/kubernetes/graph_builder/graph_components/LabelSelectorEdgeBuilder.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from checkov.kubernetes.graph_builder.graph_components.K8SEdgeBuilder import K8SEdgeBuilder
+from checkov.kubernetes.graph_builder.graph_components.blocks import KubernetesBlock
+
+
+class LabelSelectorEdgeBuilder(K8SEdgeBuilder):
+
+    @staticmethod
+    def should_search_for_edges(vertex: KubernetesBlock) -> bool:
+        if vertex.metadata.labels is not None:
```

Comment:
can `metadata` be `None`?

Reply:
no, it's initialized when [constructing](https://github.com/bridgecrewio/checkov/blob/master/checkov/kubernetes/graph_builder/local_graph.py#L53) the vertex